Alarms.h

```diff
@@ -319,13 +335,24 @@ class AlarmManager {
   int led_pin_;
   utils::Pulse led_pulse_;
   Alarm alarms_[NUM_ALARMS];
+  Alarm alarmsHeader_[NUM_ALARMS-4];
+  Alarm alarmsFooter_[4];
```

Comment:
What are these?

Reply:
4 possible new knob setting confirm alerts. A constant is now defined to represent this.